rllib/env/env_runner.py

```diff
@@ -0,0 +1,174 @@
+import abc
+import platform
+from typing import Any, Dict, Optional, Union
+
+import ray
+from ray.rllib.utils.actor_manager import FaultAwareApply
+from ray.rllib.utils.annotations import ExperimentalAPI
+
+
+@ExperimentalAPI
+class EnvRunner(FaultAwareApply, metaclass=abc.ABCMeta):
```

Comment:
When doing these refactors, can you clarify which components are copy / pasted from the past APIs? For example I don't know why we need get_node_ip() API etc. for this class. Is that really needed? or is that just copy / pasted from what we had before?

Reply:
will do!

Reply:
updated the description of this PR.